tools/delete_candidate.php

```diff
@@ -65,10 +85,59 @@
 $query = "SELECT ID, Test_name, CommentID FROM flag WHERE SessionID in (" . 
          implode(" , ", $sessions) . ")";
 $instruments = $DB->pselect($query, array()); 
-
+//delete from genomic_candidate_files_rel
+$DB->delete("genomic_candidate_files_rel", array("CandID" => $DCCID));
+echo "----------------------delete genomic_candidate_files_rel--------";
+//delete from genomic_sample_candidate_rel
+$DB->delete("genomic_sample_candidate_rel", array("CandID" => $DCCID));
+echo "----------------------delete genomic_sample_candidate_rel--------";
+//delete from mri_scanner
+$DB->delete("mri_scanner", array("CandID" => $DCCID));
+echo "----------------------delete mri_scanner-------------------------";
+//delete from parameter_candidate
+$DB->delete("parameter_candidate", array("CandID" => $DCCID));
+echo "----------------------delete parameter candidate-----------------";
+//find session id
+$sessionIDs = $DB->pselect("SELECT ID 
+                FROM session 
+                WHERE candID=:candID",
+                array('candID' => $DCCID)
+                );
+
+
+//delete from media
+foreach ($sessionIDs as $sessionID) {
```

Comment:
You could use the join(',',$sessionIDs) function in php to create a string that whould be usable by the FIND_IN_SET mysql function to delete all the media reord in a single transaction.